packages/database/src/migrations/20210824231816-DistributionOfSpeciesPacmoss-modifies-data.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecordBySql, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const SPECIES = [
+  ['Anopheles', 'An_Farauti'],
+  ['Anopheles', 'An_Koliensis'],
+  ['Anopheles', 'An_Punctulatus'],
+  ['Aedes', 'Ae_Aegypti'],
+  ['Aedes', 'Ae_Albopictus'],
+  ['Aedes', 'Ae_Cooki'],
+  ['Aedes', 'Ae_Hensilli'],
+  ['Aedes', 'Ae_Marshallensis'],
+  ['Aedes', 'Ae_Polynesiensis'],
+  ['Aedes', 'Ae_Rotumae'],
+  ['Aedes', 'Ae_Scutellaris'],
+  ['Aedes', 'Ae_Vigilax'],
+  ['Culex', 'Cx_Annulirostris'],
+  ['Culex', 'Cx_Quinquefasciatus'],
+  ['Culex', 'Cx_Sitiens'],
+  ['Mansonia', 'Mn_Uniformis'],
+];
+const HIERARCHIES = ['Country', 'District'];
```

Comment:
```suggestion
const HIERARCHYLEVELS = ['Country', 'District'];
```
Optional naming suggestion just to make it a bit clearer.